.buildkite/pipelines/on_merge.yml

```diff
@@ -78,6 +78,12 @@ steps:
         - exit_status: '*'
           limit: 1
 
+  - command: '.buildkite/pipelines/e2e.yml'
```

Comment:
can we call it `on_merge_unsupported_ftrs.yml`?

Reply:
Pushed [08a77a6](https://github.com/elastic/kibana/pull/152846/commits/08a77a6fdef00d678b48a284e22e16c69546cfd8)